apm-sniffer/apm-sdk-plugin/oracle-10.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/jdbc/oracle/define/ConnectionInstrumentation.java

```diff
@@ -111,6 +110,6 @@
     }
 
     @Override protected ClassMatch enhanceClass() {
-        return NameMatch.byName(ENHANCE_CLASS);
+        return MultiClassNameMatch.byMultiClassMatch(ENHANCE_CLASS, "oracle.jdbc.driver.T4CConnection");
```

Comment:
You are using a different coding style. Do you prefer to use a static field or literal string? I am OK with either, but don't use them at the same time.